app/src/main/res/values/dimens.xml

```diff
@@ -3,6 +3,6 @@
     <dimen name="header_margin">16dp</dimen>
     <dimen name="header_text_size">32sp</dimen>
     <dimen name="sub_header_text_size">24sp</dimen>
-    <dimen name="recycler_view_height">660dp</dimen>
+    <dimen name="recycler_view_height">600dp</dimen>
```

Comment:
Remove this. Remove the `dimens` file altogether, no one uses it anymore.

Reply:
Done